packages/framework-integration-tests/integration/provider-specific/local/stop/stop.integration.ts

```diff
@@ -0,0 +1,14 @@
+//import { AWSTestHelper } from '@boostercloud/framework-provider-aws-infrastructure'
+//import { applicationName } from '../../../helper/app-helper'
+import { expect } from '../../../helper/expect'
+
+describe('After stop', () => {
+  describe('the provider', () => {
+    it('is stopped successfully', async () => {
+      //await expect(AWSTestHelper.build(applicationName())).to.be.eventually.rejectedWith(
+      //  new RegExp(`Stack with id ${applicationName()}[^\\s]+ does not exist`)
+      //)
+      expect(true).to.be.true
```

Comment:
This is temporal. Maybe we can check that the service is no longer serving /health endpoint

Reply:
Check also <THIS> comment

Reply:
done!